modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -367,6 +367,183 @@ public void testGetProps() throws Exception {
 		}
 	}
 
+	@Test
```

Comment:
Maybe, it is time to move all the tests related with categories outside this class. This class is really big right now :(.

What do you think?

Reply:
I agree, but what do you think if I create an extra LPS for the refactor? With this and the journal articles creation

Reply:
I don't know, do you think this is a readable class right now?

Reply:
definitely not! And I want to refactor, my question is more if I should do it in this ticket or in another one

Reply:
I trust you to do what it is best for the story and the code.

Reply:
perfect then, thanks @cgoncas, I'm going to apply your suggestion to the current story and create a tech ticket to apply it to all the class (creating an extra class for the ugly testGetProp)

Reply:
Hi @cgoncas , I've created a tech task to do it https://issues.liferay.com/browse/LPS-126971